### PR TITLE
Sword of Light and Shadow: auto-return selected card to hand

### PR DESCRIPTION
**Pull request by Ebola16**
[[Sword of Light and Shadow]]

Selects up to one creature card. I don't see a reason to select a creature and then not return it to the hand so I think we can skip the return to hand prompt.

### CODE REVIEW

**Comment by JayDi85** [comment]
[[Sword of Light and Shadow]]

File: Mage.Sets/src/mage/cards/s/SwordOfLightAndShadow.java [hunk 1]
```diff
@@ -82,8 +82,11 @@ public SwordOfLightAndShadowEffect copy() {
     public boolean apply(Game game, Ability source) {
         Player controller = game.getPlayer(source.getControllerId());
         Card card = game.getCard(targetPointer.getFirst(game, source));
-        return controller != null && card != null && controller.chooseUse(
-                outcome, "Return " + card.getName() + " from your graveyard to your hand?", source, game
-        ) && controller.moveCards(card, Zone.HAND, source, game);
+
+        if (card != null) {
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/s/SwordOfLightAndShadow.java:86]: Miss controller null check

**Review by JayDi85 — CHANGES_REQUESTED** [review]

**Comment by Ebola16** [comment]
Fixed. I realize `moveCards` has its own null checks so we can just run it.

**Comment by JayDi85** [comment]
Null check is a standard code style for cards/permanents/players, do not ignore it cause it can cause problems with copy-pasting code.

**Pull request merged**